node/tests/ServerModules.test.ts

```diff
@@ -2809,83 +2939,296 @@ describe("Server Module Tests", () => {
                         nb_of_votes: isResp3 ? 559490.0 : "559490",
                         avg_rating: isResp3 ? 9.0 : "9",
                     },
+                ];
+                expect(
+                    aggreg
+                        .map(convertGlideRecordToRecord)
+                        // elements (records in array) could be reordered
+                        .sort((a, b) => (a["genre"]! > b["genre"]! ? 1 : -1)),
+                ).toEqual(expectedAggreg);
+
+                const aggregProfile: [
+                    FtAggregateReturnType,
+                    Record<string, number>,
+                ] = await GlideFt.profileAggregate(
+                    client,
+                    indexMovies,
+                    query,
+                    options,
+                );
+                expect(Object.keys(aggregProfile[1])).toEqual([
+                    "parse.time",
+                    "all.count",
+                    "sync.time",
+                    "query.time",
+                    "fetch.time",
+                    "query.count",
+                    "stage.count",
+                    "apply.time",
+                    "apply.count",
+                    "groupby.time",
+                    "groupby.count",
+                    "sortby.time",
+                    "sortby.count",
+                    "result.time",
+                    "result.count",
                 ]);
+                expect(
+                    aggregProfile[0]
+                        .map(convertGlideRecordToRecord)
+                        // elements (records in array) could be reordered
+                        .sort((a, b) => (a["genre"]! > b["genre"]! ? 1 : -1)),
+                ).toEqual(expectedAggreg);
 
                 await GlideFt.dropindex(client, indexMovies);
-                await GlideFt.dropindex(client, indexBicycles);
             },
         );
 
         it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
```

Comment:
You can move `each` to `describe` block

Reply:
For some tests, we don't have both protocols.  So it's better to keep it here.

Reply:
You can unite ones with both protocol under `describe` block if you wish